### ISSUE TITLE
Customizing different shapes

### ISSUE THREAD

**Issue by shann1231** [bumbeishvili/org-chart#339]
Hi, is it possible to add in different shapes that represent different entity types? like corporate = circle, partnership = triangle?

**Comment by bumbeishvili** [comment]
Yes, you control the width & height of each node as well as innerHTML of content, so you are not limited in that
